CMakeLists.txt

```diff
@@ -14,17 +14,38 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/module")
 # Enable needed modules and languages
 enable_language(Fortran)
 enable_testing()
+
+find_package(Java 8 REQUIRED)
+# Warning!!! After finding the first version, find_package(Java) does NOT look for others. CMAKE_FIND_PACKAGE_SORT_ORDER
+# and CMAKE_FIND_PACKAGE_SORT_DIRECTION have no effect. To override the "default" java version, set cmake variable
+# JAVA_HOME to desired java installation path.
+
+if(Java_VERSION_MAJOR GREATER 13)
+    message(FATAL_ERROR "Java ${Java_VERSION_MAJOR} is not supported")
+endif()
+
 find_package(Ant 1.7.1 REQUIRED)
-find_package(Java 1.8 REQUIRED)
+find_package(Python REQUIRED)
```

Comment:
Why do we need to introduce python dependency? That's one new dependency that doesn't make sense to me. This kind of change should be discussed with people using the tool. No?

Reply:
I think we need it to execute some new command

Reply:
This is not a runtime dependency, so the users should not be affected.

Additionally, it is scheduled to be removed in the follow-up PR (_before_ the next release). For the moment it is a hack, to insure that the build is still working while I work on finishing the changes.

Reply:
Evn if it is not runtime I find it weird to introduce python in the build where it was not needed so far.

Reply:
just FYI, there are a few other changes comming. To avoid having a too big PR some changes where left over. @clementval is this a no go from your point of view ?

Reply:
Well as I mentioned in an other comment. Big changes like moving from a bash driver to a Python driver (if it is what you are looking for) should have been discussed before as it was mentioned in the community guidelines we put in place a while back.